src/track/beatutils.cpp

```diff
@@ -389,12 +312,13 @@ double BeatUtils::computeSampleMedian(QList<double> sortedItems) {
 
 QList<double> BeatUtils::computeWindowedBpmsAndFrequencyHistogram(
         const QVector<double> beats, int windowSize, const int windowStep,
-        const int sampleRate, QMap<double, int>* frequencyHistogram) {
-    QList<double> averageBpmList;
+        const int sampleRate, QMap<double, int>* pFrequencyHistogram) {
+    DEBUG_ASSERT(pFrequencyHistogram != NULL);
```

Comment:
```suggestion
    DEBUG_ASSERT(pFrequencyHistogram);
```
`NULL` is a legacy hack from before C++11. You could use `nullptr` instead, but that's not even necessary in this case.